classificators/src/main/java/ru/okpdmarket/controller/VersionController.java

```diff
@@ -0,0 +1,25 @@
+package ru.okpdmarket.controller;
+
+import static org.springframework.util.MimeTypeUtils.APPLICATION_JSON_VALUE;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import ru.okpdmarket.util.ComponentVersion;
+
+@RestController
+public class VersionController {
+
+    private static final Map<String, String> VERSION = new HashMap<String, String>() {{
+        put("version", ComponentVersion.getComponentVersion(getClass()));
```

Comment:
Let's add  put("buildTimestamp", getComponentTitle(getClass())); and appropriate build Timestamp?
This can help us in case we forget to bump version.

Reply:
Let's also add mockMVC test to document this endpoint

Reply:
Done